terraform/gitops/stateful-resources/stateful-resources-config.tf

```diff
@@ -135,13 +140,16 @@ locals {
     stateful_resources_sync_wave = var.stateful_resources_sync_wave
     stateful_resources_name      = local.stateful_resources_name
   }
+
   all_logical_extra_namespaces  = flatten([for stateful_resource in local.stateful_resources : try(stateful_resource.logical_service_config.secret_extra_namespaces, "")])
   all_local_extra_namespaces    = flatten([for stateful_resource in local.stateful_resources : try(stateful_resource.secret_config.generate_secret_extra_namespaces, "")])
   all_local_helm_namespaces     = distinct([for stateful_resource in local.helm_stateful_resources : try(stateful_resource.local_helm_config.resource_namespace, "")])
   all_local_op_namespaces       = distinct([for stateful_resource in local.operator_stateful_resources : try(stateful_resource.local_operator_config.resource_namespace, "")])
-  
+
   percona_credentials_secret_provider_key = "minio_percona_password"
   percona_credentials_id_provider_key     = "minio_percona_username"
+
+  strimzi_kafka_grafana_dashboards_version = "0.41.0"
```

Comment:
this needs to be looked at.  it would be ideal if we could configure the dashboards on the operator instead of in each cluster object.